RoddyCore/src/de/dkfz/roddy/execution/io/LocalExecutionService.groovy

```diff
@@ -72,6 +74,31 @@ public class LocalExecutionService extends ExecutionService {
         return outputStream;
     }
 
+    /**
```

Comment:
I don't think, this is the right place for it. Please consider to move it to RuntimeService or ExecutionContext.